Greek – Accessibility Principles

remibetin opened the pull request:
Re-creates @Xrisanthi-Mil's https://github.com/w3c/wai-accessibility-principles/pull/48

Preview: https://deploy-preview-841--wai-website.netlify.app/fundamentals/accessibility-principles/el

Resolves https://github.com/w3c/wai-translations/issues/135

Xrisanthi-Mil committed:
Greek – Accessibility Principles

remibetin committed:
Merge branch 'main' into el-accessibility-principles

remibetin committed:
Update translation date and add translation metadata

remibetin committed:
Mark translation as outdated

remibetin committed:
Fix broken link

remibetin commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+### Εναλλακτικά κείμενα για μη κειμενικό περιεχόμενο {#alternatives}
+
+Τα εναλλακτικά κείμενα είναι ισοδύναμα για το μη κειμενικό περιεχόμενο. Τα παραδείγματα περιλαμβάνουν.
```
Test comment

Xrisanthi-Mil replied:
**Final**
Τα εναλλακτικά κείμενα είναι ισοδύναμα για το μη κειμενικό περιεχόμενο. Τα παραδείγματα περιλαμβάνουν:

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/pull/841/commits/ce4d460606980159b83333712f581631fab8cf18

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+{% include box.html type="start" h="2" title="Περίληψη" class="full" %}
+{:/}
+
+Αυτή η σελίδα εισάγει ορισμένες από τις απαιτήσεις προσβασιμότητας στον Παγκόσμιο Ιστό για ιστότοπους, εφαρμογές Παγκοσμίου Ιστού, προγράμματα περιήγησης και άλλα εργαλεία. Παρέχει αναφορές στα διεθνή πρότυπα της Πρωτοβουλίας για τον Παγκόσμιο Ιστό του W3C (WAI) και σε [ιστορίες χρηστών του Παγκοσμίου Ιστού](/people-use-web/user-stories/).
```
Αυτή η σελίδα εισάγει ορισμένες απο τις απαιτήσεις προσβασιμότητας για ιστοτόπους (websites), εφαρμογές (apps), προγράμματα περιήγησης (browsers) και άλλα εργαλεία του Παγκόσμιου Ιστού. Αναφέρεται στα διεθνή πρότυπα του W3C Web Accessibility Initiative (WAI), καθώς και σε ιστορίες χρηστών του ιστού.

Xrisanthi-Mil replied:
Αυτή η σελίδα εισάγει ορισμένες απο τις απαιτήσεις προσβασιμότητας στον ιστό για ιστοτόπους, διαδικτυακές εφαρμογές, προγράμματα περιήγησης και άλλα εργαλεία. Παρέχει αναφορές στα διεθνή πρότυπα της Πρωτοβουλίας για την Προσβασιμότητα στον Ιστό (WAI) του W3C και σε ιστορίες χρηστών του ιστού.

avourliotakis replied:
@Xrisanthi-Mil 
- το "στον ιστο" προτίμησα να το βάλω στο τέλος της πρώτης πρότασης, καθώς έτσι είναι πιο ευανάγνωστο. Με τα υπόλοιπα συμφωνώ.

Xrisanthi-Mil replied:
Ok. Let's agree on that. So the final translation is the following:

Αυτή η σελίδα εισάγει ορισμένες απο τις απαιτήσεις προσβασιμότητας για ιστοτόπους, διαδικτυακές εφαρμογές, προγράμματα περιήγησης και άλλα εργαλεία στον ιστό . Παρέχει αναφορές στα διεθνή πρότυπα της Πρωτοβουλίας για την Προσβασιμότητα στον Ιστό (WAI) του W3C και σε ιστορίες χρηστών του ιστού.

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/pull/841/commits/7d3d5b3e4d8b098d40e46051e9bf60228383efe9

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+## Πρότυπα προσβασιμότητας στον Παγκόσμιο Ιστό {#standards}
+
+Η προσβασιμότητα στον Παγκόσμιο Ιστό βασίζεται σε πολλά συστατικά που συνεργάζονται μεταξύ τους. Μερικά από αυτά περιλαμβάνουν:
```
Η προσβασιμότητα στον ιστό εξαρτάται απο τη συνεργασία διάφορων στοιχείων. Ορισμένα απο αυτά περιλαμβάνουν:

Xrisanthi-Mil replied:
Η προσβασιμότητα στον ιστό βασίζεται σε διάφορα στοιχεία που συνεργάζονται μεταξύ τους. Ορισμένα από αυτά περιλαμβάνουν:

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/pull/841/commits/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+-   **[Οδηγίες προσβασιμότητας των πρακτόρων χρήστη (UAAG)](/standards-guidelines/uaag/)**
+-   **[Οδηγίες προσβασιμότητας των εργαλείων συγγραφής (ATAG)](/standards-guidelines/atag/)**
+
+Υπάρχει επίσης μία προδιαγραφή του WAI για τις **[Προσβάσιμες εμπλουτισμένες εφαρμογές διαδικτύου(WAI-ARIA)](/standards-guidelines/aria/)**, που περιλαμβάνουν δυναμικό περιεχόμενο και προηγμένα στοιχεία ελέγχου διεπαφής χρήστη ανεπτυγμένα με Ajax, JavaScript και σχετικές τεχνολογίες Παγκοσμίου Ιστού.
```
Επίσης το WAI έχει ορίσει το πλαίσιο για Προσβάσιμες Πλούσιες Εφαρμογές Διαδικτύου (WAI-ARIA), το οποίο περιλαμβάνει αφενός ένα δυναμικό περιεχόμενο και αφετέρου τα στοιχεία ελέγχου για προηγμένη διεπαφή χρήστη (user interface). Αυτά συνήθως αναπτύσσονται με Ajax, JavaScript, και άλλες σχετικές τεχνολογίες ιστού.

Xrisanthi-Mil replied:
Υπάρχει επίσης μία προδιαγραφή του WAI για τις **[Προσβάσιμες Πλούσιες Εφαρμογές Διαδικτύου(WAI-ARIA)](/standards-guidelines/aria/)**, το οποίο περιλαμβάνει δυναμικό περιεχόμενο και προηγμένα στοιχεία ελέγχου διεπαφής χρήστη ανεπτυγμένα με Ajax, JavaScript και σχετικές τεχνολογίες Παγκοσμίου Ιστού.

avourliotakis replied:
"η οποία περιλαμβάνει"

Xrisanthi-Mil replied:
Accepted so the final translation is:

Υπάρχει επίσης μία προδιαγραφή του WAI για τις [Προσβάσιμες Πλούσιες Εφαρμογές Διαδικτύου(WAI-ARIA)](https://github.com/standards-guidelines/aria/), η οποία περιλαμβάνει δυναμικό περιεχόμενο και προηγμένα στοιχεία ελέγχου διεπαφής χρήστη ανεπτυγμένα με Ajax, JavaScript και σχετικές τεχνολογίες Παγκοσμίου Ιστού.

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/7d3d5b3e4d8b098d40e46051e9bf60228383efe9

Please let me know if something is wrong.

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+-   **Περιεχόμενο Παγκοσμίου Ιστού**{:#webcontent} - αναφέρεται σε κάθε τμήμα εντός ιστοτόπου, συμπεριλαμβανομένων του κειμένου, των εικόνων, των φορμών και των πολυμέσων, όπως επίσης οποιουδήποτε κώδικα σήμανσης, σενάρια, εφαρμογές και άλλα παρόμοια.
+-   **Πράκτορες χρήστη**{:#useragents} - λογισμικό που οι άνθρωποι χρησιμοποιούν για να έχουν πρόσβαση στο περιεχόμενο του Παγκοσμίου Ιστού, συμπεριλαμβανομένων προγράμματα περιήγησης γραφικών για επιτραπέζιους υπολογιστές, προγράμματα περιήγησης μέσω φωνής, προγράμματα περιήγησης κινητών τηλεφώνων, προγράμματα αναπαραγωγής πολυμέσων, πρόσθετα και μερικές [υποστηρικτικές τεχνολογίες](/people-use- ...
+-   **Εργαλεία συγγραφής**{:#authoringtools} - λογισμικό ή υπηρεσίες που χρησιμοποιούν οι άνθρωποι για την παραγωγή περιεχομένου Παγκοσμίου Ιστού, συμπεριλαμβανομένων προγραμμάτων επεξεργασίας κώδικα, εργαλείων μετατροπής εγγράφων, συστημάτων διαχείρισης περιεχομένου, ιστολογίων, σεναρίων βάσεων δεδομένων και άλλων εργαλείων.
```
**final**
Εργαλεία συγγραφής (Authoring tools) - το λογισμικό ή οι υπηρεσίες που χρησιμοποιούν οι άνθρωποι για την παραγωγή περιεχομένου στον ιστό, συμπεριλαμβανομένων των προγραμμάτων επεξεργασίας κώδικα, των εργαλείων μετατροπής εγγράφων, των συστημάτων διαχείριστης περιεχομένου, των ιστολογίων (blogs), των σεναριών (scripts) για βάσεις δεδομένων, καθώς και άλλων εργαλείων.

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+Η προσβασιμότητα στον Παγκόσμιο Ιστό βασίζεται σε πολλά συστατικά που συνεργάζονται μεταξύ τους. Μερικά από αυτά περιλαμβάνουν:
+
+-   **Περιεχόμενο Παγκοσμίου Ιστού**{:#webcontent} - αναφέρεται σε κάθε τμήμα εντός ιστοτόπου, συμπεριλαμβανομένων του κειμένου, των εικόνων, των φορμών και των πολυμέσων, όπως επίσης οποιουδήποτε κώδικα σήμανσης, σενάρια, εφαρμογές και άλλα παρόμοια.
```
**final**
Περιεχόμενο ιστού - αναφέρεται σε οποιοδήποτε τμήμα ενός ιστοτόπου που περιλαμβάνει κείμενο, εικόνες, φόρμες, πολυμέσα, καθώς και σε οτιδήποτε αφορά κώδικα σήμανσης (markup code), σενάρια (scripts), εφαρμογές (applications) και άλλα.

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+  
+Η Πρωτοβουλία για τον Παγκόσμιο Ιστό του W3C (WAI) παρέχει ένα σύνολο οδηγιών που αναγνωρίζονται διεθνώς ως το πρότυπο για την προσβασιμότητα στον Παγκόσμιο Ιστό. Αυτά περιλαμβάνουν:
+
+-   **[Οδηγίες προσβασιμότητας του περιεχομένου του Παγκοσμίου Ιστού (WCAG)](/standards-guidelines/wcag/)**
```
accepted

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/pull/841/commits/39435ce95140e89b60077907ac8bcdd975820123.

Please let me know if something is wrong.

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+-   **[Οδηγίες προσβασιμότητας του περιεχομένου του Παγκοσμίου Ιστού (WCAG)](/standards-guidelines/wcag/)**
+-   **[Οδηγίες προσβασιμότητας των πρακτόρων χρήστη (UAAG)](/standards-guidelines/uaag/)**
+-   **[Οδηγίες προσβασιμότητας των εργαλείων συγγραφής (ATAG)](/standards-guidelines/atag/)**
```
Accept the change

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/pull/841/commits/39435ce95140e89b60077907ac8bcdd975820123.

Please let me know if something is wrong.

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+Η Πρωτοβουλία για τον Παγκόσμιο Ιστό του W3C (WAI) παρέχει ένα σύνολο οδηγιών που αναγνωρίζονται διεθνώς ως το πρότυπο για την προσβασιμότητα στον Παγκόσμιο Ιστό. Αυτά περιλαμβάνουν:
+
+-   **[Οδηγίες προσβασιμότητας του περιεχομένου του Παγκοσμίου Ιστού (WCAG)](/standards-guidelines/wcag/)**
+-   **[Οδηγίες προσβασιμότητας των πρακτόρων χρήστη (UAAG)](/standards-guidelines/uaag/)**
```
Οδηγίες για την Προσβασιμότητα Πρακτόρων Χρήστη (UAAG)

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/ebfe9af0a5a8cf971f8a8cca9303263c6c866674

Please let me know if something is wrong.

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+-   Σύντομα ισοδύναμα για εικόνες, συμπεριλαμβανομένων εικονιδίων, κουμπιών και γραφικών
+-   Περιγραφή δεδομένων που αναπαρίστανται σε γραφήματα, διαγράμματα και απεικονίσεις
+-   Σύντομες περιγραφές μη κειμενικού περιεχομένου, όπως αρχεία ήχου και βίντεο
+-   Ετικέτες για στοιχεία ελέγχου φορμών, εισαγωγή δεδομένων
```
Accept the change

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/pull/841/commits/39435ce95140e89b60077907ac8bcdd975820123.

Please let me know if something is wrong.

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+## Αντιληπτή πληροφορία και διεπαφή χρήστη{#perceivable}
+
+### Εναλλακτικά κείμενα για μη κειμενικό περιεχόμενο {#alternatives}
```
the Dash might be difficult by web browsers to be read.

remibetin replied:
@Xrisanthi-Mil @avourliotakis I'm note sure if you've reached an agreement on this part of the translation. If so, could you provide the final translation? Thanks!

Xrisanthi-Mil replied:
The final translation is:

Εναλλακτικά κείμενα για μη κειμενικό περιεχόμενο {#alternatives}

remibetin replied:
@Xrisanthi-Mil Thanks! Since this is the current wording, I'm resolving this comment.

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+Τα εναλλακτικά κείμενα είναι ισοδύναμα για το μη κειμενικό περιεχόμενο. Τα παραδείγματα περιλαμβάνουν.
+
+-   Σύντομα ισοδύναμα για εικόνες, συμπεριλαμβανομένων εικονιδίων, κουμπιών και γραφικών
```
-   Σύντομα ισοδύναμα για εικόνες, συμπεριλαμβανομένων εικονιδίων, κουμπιών και γραφικών

It is translated according to the original translation

avourliotakis replied:
- Συντομες ισοδύναμες (περιγραφές) για εικόνες, συμπεριλαμβανομένων των εικονιδιών, των κουμπιών και των γραφικών. 

Σημείωση: εδώ το "equivalents" αφορά τις περιγραφες και η συμπληρωματική παρένθεση βοηθά αρκετά στην κατανόηση.

Xrisanthi-Mil replied:
Η λέξη περιγραφές είναι πρόσθετη και δεν υπάρχει στο αρχικό κείμενο. Θεωρώ ότι με την προσθήκη παρεμβαίνουμε στο αρχικό κείμενο.

Xrisanthi-Mil replied:
**Final**
Τα εναλλακτικά κείμενα είναι ισοδύναμα για το μη κειμενικό περιεχόμενο. Τα παραδείγματα περιλαμβάνουν.

-   Σύντομα ισοδύναμα για εικόνες, συμπεριλαμβανομένων εικονιδίων, κουμπιών και γραφικών

remibetin replied:
Thanks, @Xrisanthi-Mil and @avourliotakis. I understand no changes were made to this line. 

Please let me know if I missed a requested change.

avourliotakis replied:
No changes. Here's the final:

Τα εναλλακτικά κείμενα είναι ισοδύναμα για το μη κειμενικό περιεχόμενο. Τα παραδείγματα περιλαμβάνουν:

- Σύντομα ισοδύναμα για εικόνες, συμπεριλαμβανομένων εικονιδίων, κουμπιών και γραφικών

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+Τα εναλλακτικά κείμενα είναι ισοδύναμα για το μη κειμενικό περιεχόμενο. Τα παραδείγματα περιλαμβάνουν.
+
+-   Σύντομα ισοδύναμα για εικόνες, συμπεριλαμβανομένων εικονιδίων, κουμπιών και γραφικών
+-   Περιγραφή δεδομένων που αναπαρίστανται σε γραφήματα, διαγράμματα και απεικονίσεις
```
Περιγραφή δεδομένων που παρουσιάζονται σε γραφήματα, διαγράμματα και εικονογραφήσεις

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/ebfe9af0a5a8cf971f8a8cca9303263c6c866674

Please let me know if something is wrong.

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+-   Σύντομα ισοδύναμα για εικόνες, συμπεριλαμβανομένων εικονιδίων, κουμπιών και γραφικών
+-   Περιγραφή δεδομένων που αναπαρίστανται σε γραφήματα, διαγράμματα και απεικονίσεις
+-   Σύντομες περιγραφές μη κειμενικού περιεχομένου, όπως αρχεία ήχου και βίντεο
```
Σύντομες περιγραφές για μη κειμενικό περιεχόμενο, όπως είναι τα αρχεία ήχου και βίντεο

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/pull/841/commits/ebfe9af0a5a8cf971f8a8cca9303263c6c866674

Please let me know if something is wrong.

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+Τα πρότυπα διαδραματίζουν έναν κρίσιμο ρόλο στον καθορισμό των απαιτήσεων προσβασιμότητας για κάθε ένα από αυτά τα συστατικά. Κάποιες απαιτήσεις προσβασιμότητας είναι εύκολο να ικανοποιηθούν, ωστόσο η κατανόηση των βασικών για το πως οι άνθρωποι με αναπηρίες χρησιμοποιούν τον Παγκόσμιο Ιστό βοηθά στο να εφαρμοστούν πιο αποτελεσματικά και αποδοτικά. Ορισμένες πτυχές της προσβασιμότητας απαιτούν πε ...
+  
+Η Πρωτοβουλία για τον Παγκόσμιο Ιστό του W3C (WAI) παρέχει ένα σύνολο οδηγιών που αναγνωρίζονται διεθνώς ως το πρότυπο για την προσβασιμότητα στον Παγκόσμιο Ιστό. Αυτά περιλαμβάνουν:
```
Η Πρωτοβουλία για την Προσβασιμότητα του Ιστού του W3C (WAI) παρέχει ένα σύνολο κατευθυντήριων οδηγιών που αναγνωρίζεται διεθνώς ως το πρότυπο για την προσβασιμότητα στον ιστό. Σε αυτές περιλαμβάνονται:

avourliotakis replied:
"που αναγνωρίζονται διεθνώς"

Xrisanthi-Mil replied:
το υποκείμενο είναι το ένα σύνολο. Συνεπώς δε θα έπρεπε να μπει αναγνωρίζεται;

avourliotakis replied:
Σωστά!

remibetin replied:
@Xrisanthi-Mil @avourliotakis Have you reached an agreement on this part of the translation? If so, could you provide the final translation? Thanks!

Xrisanthi-Mil replied:
Η Πρωτοβουλία για την Προσβασιμότητα του Ιστού του W3C (WAI) παρέχει ένα σύνολο κατευθυντήριων οδηγιών που αναγνωρίζεται διεθνώς ως το πρότυπο για την προσβασιμότητα στον ιστό. Σε αυτές περιλαμβάνονται:

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/01b1c7487ee853a6727cb1d743da7ab3027abed3

Please let me know if something is wrong.

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+Αυτά τα στοιχεία αλληλοσυνδέονται και υποστηρίζουν το ένα το άλλο. Για παράδειγμα, το **περιεχόμενο Παγκοσμίου Ιστού** πρέπει να περιλαμβάνει εναλλακτικά κείμενα για εικόνες. Αυτή η πληροφορία πρέπει να υποβληθεί για επεξεργασία από **προγράμματα περιήγησης Παγκοσμίου Ιστού** και στη συνέχεια να μεταφερθεί σε **υποστηρικτικές τεχνολογίες**, όπως αναγνώστες οθόνης. Για να δημιουργήσουν τέτοια εναλ ...
+
+Τα πρότυπα διαδραματίζουν έναν κρίσιμο ρόλο στον καθορισμό των απαιτήσεων προσβασιμότητας για κάθε ένα από αυτά τα συστατικά. Κάποιες απαιτήσεις προσβασιμότητας είναι εύκολο να ικανοποιηθούν, ωστόσο η κατανόηση των βασικών για το πως οι άνθρωποι με αναπηρίες χρησιμοποιούν τον Παγκόσμιο Ιστό βοηθά στο να εφαρμοστούν πιο αποτελεσματικά και αποδοτικά. Ορισμένες πτυχές της προσβασιμότητας απαιτούν πε ...
```
Τα πρότυπα διαδραματίζουν έναν κρίσιμο ρόλο στον καθορισμό των απαιτήσεων προσβασιμότητας για κάθε ένα από αυτά τα στοιχεία. Ορισμένες απαιτήσεις προσβασιμότητας είναι εύκολο να ικανοποιηθούν, ωστόσο η κατανόηση των βασικών αρχών για το πως οι άνθρωποι με αναπηρίες χρησιμοποιούν τον Ιστό βοηθά στο να εφαρμοστούν πιο αποτελεσματικά και αποδοτικά. Ορισμένες πτυχές της προσβασιμότητας απαιτούν περισσότερες τεχνικές δεξιότητες ή προηγμένες γνώσεις για τον τρόπο με τον οποίο οι άνθρωποι χρησιμοποιούν τον Ιστό. Σε κάθε περίπτωση, [η συμμετοχή χρηστών από τα πρώτα στάδια και καθ' όλη τη διάρκεια των διαδικτυακών σας έργων](/test-evaluate/involving-users/) θα κάνει τη δουλειά σας καλύτερη και ευκολότερη.

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+{% include excol.html type="middle" %}
+
+Αυτά τα στοιχεία αλληλοσυνδέονται και υποστηρίζουν το ένα το άλλο. Για παράδειγμα, το **περιεχόμενο Παγκοσμίου Ιστού** πρέπει να περιλαμβάνει εναλλακτικά κείμενα για εικόνες. Αυτή η πληροφορία πρέπει να υποβληθεί για επεξεργασία από **προγράμματα περιήγησης Παγκοσμίου Ιστού** και στη συνέχεια να μεταφερθεί σε **υποστηρικτικές τεχνολογίες**, όπως αναγνώστες οθόνης. Για να δημιουργήσουν τέτοια εναλ ...
```
Accepted

remibetin replied:
@avourliotakis @Xrisanthi-Mil Could you provide the full paragraph with modifications? I will then apply the change to the branch.

Xrisanthi-Mil replied:
Αυτά τα στοιχεία αλληλοσυνδέονται και υποστηρίζουν το ένα το άλλο. Για παράδειγμα, το **περιεχόμενο του ιστού** πρέπει να περιλαμβάνει εναλλακτικά κείμενα για εικόνες. Αυτή η πληροφορία πρέπει να υποβληθεί σε επεξεργασία από τα **προγράμματα περιήγησης ** και στη συνέχεια να μεταφερθεί σε **υποστηρικτικές τεχνολογίες**, όπως σε αναγνώστες οθόνης. Για να δημιουργήσουν τέτοια εναλλακτικά κείμενα, οι συγγραφείς χρειάζονται τα **εργαλεία συγγραφής** που τους υποστηρίζουν να το κάνουν. Περισσότερες πληροφορίες παρέχονται στα [[Βασικά συστατικά για την Προσβασιμότητα στον Ιστό]](/fundamentals/components/)

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/7d3d5b3e4d8b098d40e46051e9bf60228383efe9

Please let me know if something is wrong.

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+{% include excol.html type="start" %}
+
+### Περισσότερα σχετικά με τα πρότυπα προσβασιμότητας Παγκοσμίου Iστού
```
Accept the change

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/pull/841/commits/39435ce95140e89b60077907ac8bcdd975820123.

Please let me know if something is wrong.

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+Η προσβασιμότητα στον Παγκόσμιο Ιστό βασίζεται σε πολλά συστατικά που συνεργάζονται μεταξύ τους. Μερικά από αυτά περιλαμβάνουν:
+
+-   **Περιεχόμενο Παγκοσμίου Ιστού**{:#webcontent} - αναφέρεται σε κάθε τμήμα εντός ιστοτόπου, συμπεριλαμβανομένων του κειμένου, των εικόνων, των φορμών και των πολυμέσων, όπως επίσης οποιουδήποτε κώδικα σήμανσης, σενάρια, εφαρμογές και άλλα παρόμοια.
+-   **Πράκτορες χρήστη**{:#useragents} - λογισμικό που οι άνθρωποι χρησιμοποιούν για να έχουν πρόσβαση στο περιεχόμενο του Παγκοσμίου Ιστού, συμπεριλαμβανομένων προγράμματα περιήγησης γραφικών για επιτραπέζιους υπολογιστές, προγράμματα περιήγησης μέσω φωνής, προγράμματα περιήγησης κινητών τηλεφώνων, προγράμματα αναπαραγωγής πολυμέσων, πρόσθετα και μερικές [υποστηρικτικές τεχνολογίες](/people-use- ...
```
Πράκτορες χρήστη (user agents) - το λογισμικό που χρησιμοποιούν οι άνθρωποι για να έχουν πρόσβαση στο περιεχόμενο του ιστού, συμπεριλαμβανομένων των προγραμμάτων περιηγησης γραφικών για επιτραπέζιους υπολογιστές, των προγραμμάτων περιήγησης μέσω φωνής, των προγραμμάτων περιήγησης για κινητά τηλέφωνα, των προγραμμάτων αναπαραγωγής πολυμέσων, των πρόσθετων και ορισμένων υποστηρικτικών τεχνολογιών.

avourliotakis replied:
Συμφωνώ με το "πράκτορες χρήστη".

Σημείωση: Οι παρενθέσεις έχουν συμπληρωματικό χαρακτήρα και στοχεύουν στην καλύτερη κατανόηση του περιεχομένου απο τους αναγνώστες. Πολλές λέξεις (κυρίως που αφορούν το web) συνηθίζουμε να τις λέμε στα αγγλικά, όπως πχ. plug-ins, browsers. Αν γίνεται να τις χρησιμοποιούμε συνδυαστικά, όπου χρειάζεται, πιστεύω θα βοηθήσει.

Xrisanthi-Mil replied:
Maybe @remibetin can help us on that.
Should we provide the english version of a word that is commonly used in greek too in a parenthesis. For example browsers is translated "Προγραμματα περιήγησης" in greek. 
In the translation we should keep only the greek word or add the english too in a parenthesis for example   Προγραμματα περιήγησης (browsers)?

remibetin replied:
Thanks for asking, @Xrisanthi-Mil.

> In the translation we should keep only the greek word or add the english too in a parenthesis

There is no strict guidance on this. If you expect most people will be familiar with the word in Greek, it's fine to only use Greek. If you consider that adding the English word can help understanding, it's also fine to include the English word in parenthesis.

In such case, make sure to identify the language of this part. For example:
```
Προγραμματα περιήγησης (<span lang="en">browsers</span>)
```

In case it helps, I note that in a [Greek translation of another page](https://www.w3.org/WAI/fundamentals/accessibility-intro/el), "web browsers" was added in parentheses:

> Η προσβασιμότητα στον ιστό εξαρτάται από τη συνεργασία διαφόρων στοιχείων, συμπεριλαμβανομένων των τεχνολογιών ιστού, των προγραμμάτων περιήγησης στον ιστό (web browsers ) και άλλων "user agents", των εργαλείων σύνταξης και των ιστότοπων.

Xrisanthi-Mil replied:
**Final**
Πράκτορες χρήστη (user agents) - το λογισμικό που χρησιμοποιούν οι άνθρωποι για να έχουν πρόσβαση στο περιεχόμενο του ιστού, συμπεριλαμβανομένων των προγραμμάτων περιηγησης (browsers) γραφικών για επιτραπέζιους υπολογιστές (desktops), των προγραμμάτων περιήγησης μέσω φωνής, των προγραμμάτων περιήγησης για κινητά τηλέφωνα, των προγραμμάτων αναπαραγωγής πολυμέσων, των πρόσθετων (plug-ins), και ορισμένων υποστηρικτικών τεχνολογιών (assistive technologies).

remibetin committed:
Apply suggestions by @avourliotakis accepted by @Xrisanthi-Mil

remibetin committed:
Revert one change that was not agreed on.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+Αυτή η σελίδα εισάγει ορισμένες από τις απαιτήσεις προσβασιμότητας στον Παγκόσμιο Ιστό για ιστότοπους, εφαρμογές Παγκοσμίου Ιστού, προγράμματα περιήγησης και άλλα εργαλεία. Παρέχει αναφορές στα διεθνή πρότυπα της Πρωτοβουλίας για τον Παγκόσμιο Ιστό του W3C (WAI) και σε [ιστορίες χρηστών του Παγκοσμίου Ιστού](/people-use-web/user-stories/).
+
+**Σημείωση:** Αυτή δεν αποτελεί μια πλήρης λίστα όλων των απαιτήσεων προσβασιμότητας.
```
Σημείωση: Αυτή δεν αποτελεί μια πλήρη λίστα όλων των απαιτήσεων προσβασιμότητας.

Xrisanthi-Mil replied:
Accepted

remibetin replied:
Updated, thank you both.

remibetin committed:
Merge branch 'main' into el-accessibility-principles

remibetin committed:
Apply updates discussed between @Xrisanthi-Mil and @avourliotakis

remibetin committed:
Merge branch 'el-accessibility-principles' of github.com:w3c/wai-website into el-accessibility-principles

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+-   Σύντομα ισοδύναμα για εικόνες, συμπεριλαμβανομένων εικονιδίων, κουμπιών και γραφικών
+-   Περιγραφή δεδομένων που παρουσιάζονται σε γραφήματα, διαγράμματα και εικονογραφήσεις
+-   Σύντομες περιγραφές για μη κειμενικό περιεχόμενο, όπως είναι τα αρχεία ήχου και βίντεο
+-   Οι ετικέτες για στοιχεία ελέγχου φορμών, εισαγωγής δεδομένων, καθώς και άλλων στοιχείων διεπαφής
```
Ας βγάλουμε το άρθρο "Οι" και απο εδώ για λόγους συνέπειας με τα προηγούμενα

Xrisanthi-Mil replied:
Accepted

remibetin replied:
@Xrisanthi-Mil @avourliotakis Would you please provide the updated phrase?

Xrisanthi-Mil replied:
Σύντομα ισοδύναμα για εικόνες, συμπεριλαμβανομένων εικονιδίων, κουμπιών και γραφικών
Περιγραφή δεδομένων που παρουσιάζονται σε γραφήματα, διαγράμματα και εικονογραφήσεις
Σύντομες περιγραφές για μη κειμενικό περιεχόμενο, όπως είναι τα αρχεία ήχου και βίντεο
Ετικέτες για στοιχεία ελέγχου φορμών, εισαγωγής δεδομένων, καθώς και άλλων στοιχείων διεπαφής

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/pull/841/commits/01b1c7487ee853a6727cb1d743da7ab3027abed3

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+-   [Αρχή A.1: Οι διεπαφές χρήστη του εργαλείου συγγραφής ακολουθούν τις ισχύουσες οδηγίες προσβασιμότητας](https://www.w3.org/TR/ATAG20/#principle_a1)
+-   [Κατευθυντήρια γραμμή Α.2.1: (Για τη διεπαφή χρήστη του εργαλείου συγγραφής) Διαθέστε εναλλακτικό περιεχόμενο στους συγγραφείς](https://www.w3.org/TR/ATAG20/#gl_a21)
+-   [Κατευθυντήρια γραμμή Α.2.2: (Για τη διεπαφή χρήστη του εργαλείου συγγραφής) Διασφαλίστε ότι η παρουσίαση επεξεργασίας-προβολής μπορεί να καθοριστεί μέσω προγραμματισμού](https://www.w3.org/TR/ATAG20/#gl_a22)
+-   [Κατευθυντήρια γραμμή Α.3.7: (Για τη διεπαφή χρήστη του εργαλείου συγγραφής) Διασφαλίστε ότι οι προεπισκοπήσεις είναι τουλάχιστον όσο προσβάσιμες όσο στους εντός της αγοράς πράκτορες χρήστη](https://www.w3.org/TR/ATAG20/#gl_a37)
```
"Διασφαλίστε ότι οι προεσκοπήσεις είναι τουλάχιστον το ίδιο προσβάσιμες και εντός της αγοράς των πρακτόρων χρήστη"

Xrisanthi-Mil replied:
Accepted

remibetin replied:
Updated, thank you both!

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+{% include excol.html type="start" %}
+
+#### Ιστορίες σχετικές με εναλλακτικά κείμενα
```
Πιθανά έχουν ανανεωθεί τα links απο αυτή την λίστα. Μεταφράζω τα καινούρια:

- Ade, δημοσιογράφος με περιορισμένη χρήση των χεριών του
- Lakshmi, ανώτερο στέλεχος λογίστρια που είναι τυφλή
- Dhruv, μεγαλύτερος ενήλικας μαθητής που είναι κωφός
- Marta, βοηθός μάρκετινγκ που είναι κωφή και τυφλή
- Stefan, μαθητής με διαταραχή ελλειμματικής προσοχής και υπερκινητικότητας, καθώς και με δυσλεξία
- Elias, συνταξιούχος με περιορισμένη όραση, τρέμουλο στα χέρια και ήπιου βαθμού απώλεια βραχυπρόθεσμης μνήμης

Xrisanthi-Mil replied:
**Final**
[Ade, δημοσιογράφος με περιορισμένη χρήση των χεριών του](/people-use-web/user-stories/story-one/)
[Lakshmi, ανώτερο στέλεχος - λογίστρια που είναι τυφλή](/people-use-web/user-stories/story-three/)
[Dhruv, μεγαλύτερος ενήλικας μαθητής που είναι κωφός](/people-use-web/user-stories/story-six/)
[Marta, βοηθός μάρκετινγκ που είναι κωφή και τυφλή](/people-use-web/user-stories/story-seven/)
[Stefan, μαθητής με διαταραχή ελλειμματικής προσοχής και υπερκινητικότητας, καθώς και με δυσλεξία](/people-use-web/user-stories/story-eight/)
[Elias, συνταξιούχος με περιορισμένη όραση, τρέμουλο στα χέρια και ήπιου βαθμού απώλεια βραχυπρόθεσμης μνήμης](/people-use-web/user-stories/story-nine/)

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+{% include excol.html type="end" %}
+
+### Το περιεχόμενο είναι ευκολότερο να ιδωθεί και να ακουστεί {#distinguishable}
```
Εδώ η παθητική φωνή ίσως δυσκολέψει. Αφήνω την ενεργητική εκδοχή μήπως συμφωνήσουμε σε αυτή τελικά:

- Το περιεχόμενο είναι ευκολότερο να το δείτε (ή δεις) και να το ακούσετε (ή ακούσεις)

Xrisanthi-Mil replied:
Απλά ο ενδοιασμός μου είναι ότι δεν έχει χρησιμοποιηθεί καθόλου ο β' ενικός ή β΄ πληθυντικός σε όλη τη μετάφραση. 
Να το δούμε έτσι καλύτερα:
Το περιεχόμενο είναι πιο εύκολο να το δει και να το ακούσει κανείς

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/pull/841/commits/19954f117f4cf5ae2a29501b958fe1b3d1726543

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+Το διακριτό περιεχόμενο είναι ευκολότερο να το ιδωθεί και να ακουστεί. Τέτοιο περιεχόμενο περιλαμβάνει:
+
+-   Το χρώμα δεν χρησιμοποιείται ως ο μοναδικός τρόπος μετάδοσης πληροφορίας ή αναγνώρισης περιεχομένου
```
Χρώμα που δεν χρησιμοποιείται μόνο για την μετάδοση της πληροφορίας ή αναγνώρισης περιεχομένου

Xrisanthi-Mil replied:
Το διακριτό περιεχόμενο είναι πιο εύκολο να το δει και να το ακούσει κανείς Σε αυτό το περιεχόμενο περιλαμβάνονται τα εξής:
Το χρώμα δεν χρησιμοποιείται ως ο μοναδικός τρόπος μετάδοσης πληροφορίας ή αναγνώρισης περιεχομένου

Γιατί θεωρείς ότι είναι πιο σωστό να βγει το άρθρο;

avourliotakis replied:
Μου ακούγεται πιο φυσικό με καλύτερη ροή στην ανάγνωση. Ωστόσο δεν αλλάζει σημαντικά αυτό που θέλουμε να επικοινωνήσουμε.

Xrisanthi-Mil replied:
**Final**
Το χρώμα δεν χρησιμοποιείται ως ο μοναδικός τρόπος μετάδοσης πληροφορίας ή αναγνώρισης περιεχομένου

remibetin replied:
@Xrisanthi-Mil I understand no changes were finally made to this line. Please let me know if I missed a requested change.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+Το διακριτό περιεχόμενο είναι ευκολότερο να το ιδωθεί και να ακουστεί. Τέτοιο περιεχόμενο περιλαμβάνει:
+
+-   Το χρώμα δεν χρησιμοποιείται ως ο μοναδικός τρόπος μετάδοσης πληροφορίας ή αναγνώρισης περιεχομένου
+-   Οι προεπιλεγμένοι χρωματικοί συνδυασμοί προσκηνίου και παρασκηνίου παρέχουν επαρκή αντίθεση
```
Προεπιλεγμένους χρωματικούς συνδυασμούς στο προσκήνιο και το παρασκήνιο/φόντο με επαρκή αντίθεση

Xrisanthi-Mil replied:
Γιατί θεωρείς ότι είναι πιο σωστό να βγει το άρθρο;

Xrisanthi-Mil replied:
**Final**
Οι προεπιλεγμένοι χρωματικοί συνδυασμοί προσκηνίου και παρασκηνίου παρέχουν επαρκή αντίθεση

remibetin replied:
@Xrisanthi-Mil I understand that no changes were finally made to this line. Please let me know if I missed a requested change. cc @avourliotakis

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+-   Το χρώμα δεν χρησιμοποιείται ως ο μοναδικός τρόπος μετάδοσης πληροφορίας ή αναγνώρισης περιεχομένου
+-   Οι προεπιλεγμένοι χρωματικοί συνδυασμοί προσκηνίου και παρασκηνίου παρέχουν επαρκή αντίθεση
+-   Όταν οι χρήστες τροποποιούν το μέγεθος κειμένου έως και 400% ή αλλάζουν διάστιχο, δεν υπάρχει απώλεια πληροφορίας
```
Αλλαγή μεγέθους κειμένου εώς και 400% ή αλλαγή διαστίχου απο τους χρήστες, χωρίς απώλεια της πληροφορίας

Xrisanthi-Mil replied:
Γιατί θεωρείς ότι είναι πιο σωστό να βγει το άρθρο;

Xrisanthi-Mil replied:
**final**
Όταν οι χρήστες τροποποιούν το μέγεθος κειμένου έως και 400% ή αλλάζουν διάστιχο, δεν υπάρχει απώλεια πληροφορίας

remibetin replied:
@Xrisanthi-Mil I understand that no changes were finally made to this line. Please let me know if I missed a requested change. cc @avourliotakis

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+-   Το χρώμα δεν χρησιμοποιείται ως ο μοναδικός τρόπος μετάδοσης πληροφορίας ή αναγνώρισης περιεχομένου
+-   Οι προεπιλεγμένοι χρωματικοί συνδυασμοί προσκηνίου και παρασκηνίου παρέχουν επαρκή αντίθεση
+-   Όταν οι χρήστες τροποποιούν το μέγεθος κειμένου έως και 400% ή αλλάζουν διάστιχο, δεν υπάρχει απώλεια πληροφορίας
+-   Το κείμενο επανακυκλοφορεί σε μικρά παράθυρα ("θύρες προβολής") και όταν οι χρήστες κάνουν το κείμενο μεγαλύτερο
```
Επαναπροσαρμογή κειμένου σε μικρά παράθυρα ("ορατή περιοχή") και όταν οι χρήστες μεγαλώνουν το κείμενο

Xrisanthi-Mil replied:
Γιατί θεωρείς ότι είναι πιο σωστό να βγει το άρθρο;

Xrisanthi-Mil replied:
**final**
Το κείμενο επαναπροσαρμόζεται σε μικρά παράθυρα ("ορατή περιοχή") και όταν οι χρήστες μεγαλώνουν το κείμενο

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/pull/841/commits/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+-   Οι προεπιλεγμένοι χρωματικοί συνδυασμοί προσκηνίου και παρασκηνίου παρέχουν επαρκή αντίθεση
+-   Όταν οι χρήστες τροποποιούν το μέγεθος κειμένου έως και 400% ή αλλάζουν διάστιχο, δεν υπάρχει απώλεια πληροφορίας
+-   Το κείμενο επανακυκλοφορεί σε μικρά παράθυρα ("θύρες προβολής") και όταν οι χρήστες κάνουν το κείμενο μεγαλύτερο
+-   Οι εικόνες κειμένου είναι τροποποιήσιμες ως προς το μέγεθος, αντικαθίστανται με πραγματικό κείμενο ή αποφεύγονται όπου είναι δυνατόν
```
Εικόνες κειμένου που μπορούν να αλλάξουν μέγεθος, να αντικατασταθούν με πραγματικό κείμενο ή να αποφευγονται όπου είναι δυνατόν

Xrisanthi-Mil replied:
Γιατί θεωρείς ότι είναι πιο σωστό να βγει το άρθρο;

Xrisanthi-Mil replied:
**final**
Οι εικόνες κειμένου μπορούν να αλλάξουν μέγεθος, να αντικατασταθούν με πραγματικό κείμενο ή να αποφεύγονται όπου είναι δυνατόν

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/pull/841/commits/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+-   Όταν οι χρήστες τροποποιούν το μέγεθος κειμένου έως και 400% ή αλλάζουν διάστιχο, δεν υπάρχει απώλεια πληροφορίας
+-   Το κείμενο επανακυκλοφορεί σε μικρά παράθυρα ("θύρες προβολής") και όταν οι χρήστες κάνουν το κείμενο μεγαλύτερο
+-   Οι εικόνες κειμένου είναι τροποποιήσιμες ως προς το μέγεθος, αντικαθίστανται με πραγματικό κείμενο ή αποφεύγονται όπου είναι δυνατόν
+-   Οι χρήστες μπορούν να κάνουν παύση, να σταματήσουν ή να προσαρμόσουν την ένταση του ήχου που αναπαράγεται σε έναν ιστότοπο
```
Χρήστες που μπορούν να κάνουν πάυση, να σταματήσουν ή να προσαρμόσουν την ένταση του ήχου που αναπαράγεται σε έναν ιστότοπο

Xrisanthi-Mil replied:
Γιατί θεωρείς ότι είναι πιο σωστό να βγει το άρθρο;

Xrisanthi-Mil replied:
**final**
Οι χρήστες μπορούν να κάνουν παύση, να σταματήσουν ή να προσαρμόσουν την ένταση του ήχου που αναπαράγεται σε έναν ιστότοπο

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+-   Το κείμενο επανακυκλοφορεί σε μικρά παράθυρα ("θύρες προβολής") και όταν οι χρήστες κάνουν το κείμενο μεγαλύτερο
+-   Οι εικόνες κειμένου είναι τροποποιήσιμες ως προς το μέγεθος, αντικαθίστανται με πραγματικό κείμενο ή αποφεύγονται όπου είναι δυνατόν
+-   Οι χρήστες μπορούν να κάνουν παύση, να σταματήσουν ή να προσαρμόσουν την ένταση του ήχου που αναπαράγεται σε έναν ιστότοπο
+-   Ο ήχος στο παρασκήνιο είναι χαμηλός ή μπορεί να απενεργοποιηθεί, για να αποφευχθούν παρεμβολές ή απόσπαση της προσοχής
```
Ήχο στο παρασκήνιο...

Xrisanthi-Mil replied:
**final**
Ο ήχος στο παρασκήνιο είναι χαμηλός ή μπορεί να απενεργοποιηθεί, για να αποφευχθούν παρεμβολές ή απόσπαση της προσοχής

remibetin replied:
@Xrisanthi-Mil I understand that no changes were finally made to this line. Please let me know if I missed a requested change. cc @avourliotakis

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+-   Οι αλληλουχίες πληροφορίας ή οδηγιών είναι ανεξάρτητες από οποιαδήποτε παρουσίαση
+-   Τα προγράμματα περιήγησης και οι υποστηρικτικές τεχνολογίες παρέχουν ρυθμίσεις για την προσαρμογή της παρουσίασης
+
+Η ικανοποίηση αυτής της απαίτησης επιτρέπει στο περιεχόμενο να εκφωνείται σωστά, να μεγεθύνεται ή να προσαρμόζεται ώστε να καλύπτει τις ανάγκες και τις προτιμήσεις διαφορετικών ανθρώπων. Για παράδειγμα, μπορεί να παρουσιαστεί χρησιμοποιώντας προσαρμοσμένους χρωματικούς συνδυασμούς, μέγεθος κειμένου ή άλλο στυλ για διευκόλυνση της ανάγνωσης. Αυτή η απαίτηση διευκολύνει επίσης άλλες μορφές προσαρμο ...
```
Η εκπλήρωση αυτής της απαίτησης επιτρέπει την ορθή εκφώνηση, μεγέθυνση ή προσαρμογή του περιεχομένου, ώστε να ανταποκρίνεται στις ανάγκες και τις προτιμήσεις διαφορετικών ανθρώπων. 

Για να διευκολύνει την ανάγνωση, για παράδειγμα, μπορεί να παρουσιαστεί χρησιμοποιώντας προσαρμοσμένους συνδυασμούς χρωμάτων, μεγέθος κειμένου ή κάποιας άλλης στυλιστικής επιλογής. 

Επίσης η απαίτηση αυτή διευκολύνει και άλλες μορφές προσαρμογής, συμπεριλαμβανομένης της αυτόματης δημιουργίας του περιγράμματος/δομής της σελίδας και των συνόψεων κειμένου, ώστε να βοηθήσει τους χρήστες να έχουν μια σφαιρική εικόνα και να εστιάζουν σε συγκεκριμένα τμήματα πιο εύκολα.

avourliotakis replied:
Note:
Meet an accessibility requirement = πληρώ μια απαίτηση προσβασιμότητας

Xrisanthi-Mil replied:
Η εκπλήρωση αυτής της απαίτησης επιτρέπει στο περιεχόμενο να εκφωνείται σωστά, να μεγεθύνεται ή να προσαρμόζεται ώστε να καλύπτει τις ανάγκες και τις προτιμήσεις διαφορετικών ανθρώπων. Για παράδειγμα, μπορεί να παρουσιαστεί χρησιμοποιώντας προσαρμοσμένους χρωματικούς συνδυασμούς, μέγεθος κειμένου ή άλλο στυλ για διευκόλυνση της ανάγνωσης. Αυτή η απαίτηση διευκολύνει επίσης άλλες μορφές προσαρμογής, συμπεριλαμβανομένης της αυτόματης παραγωγής περιγραμμάτων σελίδας και περιλήψεων ώστε να βοηθηθούν οι άνθρωποι να αποκτήσουν μια σφαιρική εικόνα και να εστιάσουν σε συγκεκριμένα τμήματα πιο εύκολα .

Είναι ωραίος ο τόνος που χρησιμοποιείς αλλά μήπως παρεμβαίνουμε στο αρχικό κείμενο;

avourliotakis replied:
Δεν το θεωρώ παρέμβαση αλλά καλύτερη επικοινωνία στο ελληνικό κοινό. Μπορούμε ωστόσο να το ανεβάσουμε έτσι και το αλλάζουμε μεταγενέστερα

Xrisanthi-Mil replied:
**Final**
Η εκπλήρωση αυτής της απαίτησης επιτρέπει στο περιεχόμενο να εκφωνείται σωστά, να μεγεθύνεται ή να προσαρμόζεται ώστε να καλύπτει τις ανάγκες και τις προτιμήσεις διαφορετικών ανθρώπων. Για παράδειγμα, μπορεί να παρουσιαστεί χρησιμοποιώντας προσαρμοσμένους χρωματικούς συνδυασμούς, μέγεθος κειμένου ή άλλο στυλ για διευκόλυνση της ανάγνωσης. Αυτή η απαίτηση διευκολύνει επίσης άλλες μορφές προσαρμογής, συμπεριλαμβανομένης της αυτόματης παραγωγής περιγραμμάτων σελίδας και περιλήψεων ώστε να βοηθηθούν οι άνθρωποι να αποκτήσουν μια σφαιρική εικόνα και να εστιάσουν σε συγκεκριμένα τμήματα πιο εύκολα .

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/pull/841/commits/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

remibetin committed:
Apply updates discussed between @Xrisanthi-Mil and @avourliotakis

remibetin committed:
Apply updates discussed between @Xrisanthi-Mil and @avourliotakis

remibetin committed:
Apply updates discussed between @Xrisanthi-Mil and @avourliotakis

remibetin committed:
Apply updates discussed between @Xrisanthi-Mil and @avourliotakis

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+{% include excol.html type="middle" %}
+
+-   [Martine, βαρήκοη εξ' αποστάσεως φοιτήτρια](/people-use-web/user-stories/#onlinestudent)
```
Να διορθωθούν τα links των user stories:

- Lakshmi, ανώτερο στέλεχος - λογίστρια που είναι τυφλή
- Dhruv, μεγαλύτερος ενήλικας μαθητής που είναι κωφός
- Marta, βοηθός μάρκετινγκ που είναι κωφή και τυφλή

Xrisanthi-Mil replied:
**Final**
Lakshmi, ανώτερο στέλεχος - λογίστρια που είναι τυφλή(/people-use-web/user-stories/story-three/)
Dhruv, μεγαλύτερος ενήλικας μαθητής που είναι κωφός(/people-use-web/user-stories/story-six/)
Marta, βοηθός μάρκετινγκ που είναι κωφή και τυφλή(/people-use-web/user-stories/story-seven/)

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+Για να μπορούν οι χρήστες να αλλάζουν την παρουσίαση του περιεχομένου, είναι απαραίτητο:
+
+-   Οι επικεφαλίδες, οι λίστες, οι πίνακες, τα πεδία εισαγωγής δεδομένων και οι δομές περιεχομένου να επισημαίνονται σωστά
+-   Οι αλληλουχίες πληροφορίας ή οδηγιών είναι ανεξάρτητες από οποιαδήποτε παρουσίαση
```
*να είναι

Xrisanthi-Mil replied:
**Final**
Για να μπορούν οι χρήστες να αλλάζουν την παρουσίαση του περιεχομένου, είναι απαραίτητο:

-   Οι επικεφαλίδες, οι λίστες, οι πίνακες, τα πεδία εισαγωγής δεδομένων και οι δομές περιεχομένου να επισημαίνονται σωστά
-   Οι αλληλουχίες πληροφορίας ή οδηγιών να είναι ανεξάρτητες από οποιαδήποτε παρουσίαση

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+-   Οι επικεφαλίδες, οι λίστες, οι πίνακες, τα πεδία εισαγωγής δεδομένων και οι δομές περιεχομένου να επισημαίνονται σωστά
+-   Οι αλληλουχίες πληροφορίας ή οδηγιών είναι ανεξάρτητες από οποιαδήποτε παρουσίαση
+-   Τα προγράμματα περιήγησης και οι υποστηρικτικές τεχνολογίες παρέχουν ρυθμίσεις για την προσαρμογή της παρουσίασης
```
*να παρέχουν

Xrisanthi-Mil replied:
**final**
-   Οι επικεφαλίδες, οι λίστες, οι πίνακες, τα πεδία εισαγωγής δεδομένων και οι δομές περιεχομένου να επισημαίνονται σωστά
-   Οι αλληλουχίες πληροφορίας ή οδηγιών είναι ανεξάρτητες από οποιαδήποτε παρουσίαση
-   Τα προγράμματα περιήγησης και οι υποστηρικτικές τεχνολογίες να παρέχουν ρυθμίσεις για την προσαρμογή της παρουσίασης

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+{% include excol.html type="middle" %}
+
+-   [Lee, αγοραστής μέσω διαδικτύου με αχρωματοψία](/people-use-web/user-stories/#shopper)
```
Να μπούν τα updated user stories:

- Ade, δημοσιογράφος με περιορισμένη χρήση των χεριών του
- Lakshmi, ανώτερο στέλεχος λογίστρια που είναι τυφλή
- Dhruv, μεγαλύτερος ενήλικας μαθητής που είναι κωφός
- Marta, βοηθός μάρκετινγκ που είναι κωφή και τυφλή
- Stefan, μαθητής με διαταραχή ελλειμματικής προσοχής και υπερκινητικότητας, καθώς και με δυσλεξία
- Elias, συνταξιούχος με περιορισμένη όραση, τρέμουλο στα χέρια και ήπιου βαθμού απώλεια βραχυπρόθεσμης μνήμης

Xrisanthi-Mil replied:
Accepted (user stories changed)

Xrisanthi-Mil replied:
[Ade, δημοσιογράφος με περιορισμένη χρήση των χεριών του](/people-use-web/user-stories/story-one/)
[Lakshmi, ανώτερο στέλεχος - λογίστρια που είναι τυφλή](/people-use-web/user-stories/story-three/)
[Dhruv, μεγαλύτερος ενήλικας μαθητής που είναι κωφός](/people-use-web/user-stories/story-six/)
[Marta, βοηθός μάρκετινγκ που είναι κωφή και τυφλή](/people-use-web/user-stories/story-seven/)
[Stefan, μαθητής με διαταραχή ελλειμματικής προσοχής και υπερκινητικότητας, καθώς και με δυσλεξία](/people-use-web/user-stories/story-eight/)
[Elias, συνταξιούχος με περιορισμένη όραση, τρέμουλο στα χέρια και ήπιου βαθμού απώλεια βραχυπρόθεσμης μνήμης](/people-use-web/user-stories/story-nine/)

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+-   Οι χρήστες μπορούν να κάνουν παύση, να σταματήσουν ή να προσαρμόσουν την ένταση του ήχου που αναπαράγεται σε έναν ιστότοπο
+-   Ο ήχος στο παρασκήνιο είναι χαμηλός ή μπορεί να απενεργοποιηθεί, για να αποφευχθούν παρεμβολές ή απόσπαση της προσοχής
+
+Η ικανοποίηση αυτής της απαίτησης βοηθά στο διαχωρισμό του προσκηνίου από το παρασκήνιο, ώστε η σημαντική πληροφορία να γίνει πιο διακριτή . Αυτό περιλαμβάνει σκέψεις για άτομα που δεν χρησιμοποιούν υποστηρικτικές τεχνολογίες και για άτομα χρήστες υποστηρικτικών τεχνολογιών που ενδέχεται να παρατηρήσουν παρεμβολή από εξέχων ακουστικό ή οπτικό περιεχόμενο στο παρασκήνιο. Για παράδειγμα, πολλοί άνθ ...
```
Η εκπλήρωση αυτής της απαίτησης βοηθά στον διαχωρισμό του προσκηνίου (foreground) από το φόντο (background), ώστε οι σημαντικές πληροφορίες να είναι πιο διακριτές. Αυτό περιλαμβάνει ζητήματα που αφορούν άτομα που δεν χρησιμοποιούν υποστηρικτικές τεχνολογίες και άτομα που χρησιμοποιούν υποστηρικτικές τεχνολογίες και ενδέχεται να παρατηρήσουν παρεμβολές από εμφανές ηχητικό ή οπτικό περιεχόμενο στο φόντο. Για παράδειγμα, πολλά άτομα με αχρωματοψία δεν χρησιμοποιούν συγκεκριμένα εργαλεία και βασίζονται σε έναν κατάλληλο σχεδιασμό που παρέχει επαρκή χρωματική αντίθεση μεταξύ του κειμένου και του περιβάλλοντος φόντου. Για άλλους, ο ήχος που αναπαράγεται αυτόματα θα μπορούσε να επηρεάσει τη μετατροπή κειμένου σε ομιλία ή τις συσκευές ακουστικής υποστήριξης (ALD).

Xrisanthi-Mil replied:
Accepted

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+{% include excol.html type="middle" %}
+
+-   [Lee, αγοραστής μέσω διαδικτύου με αχρωματοψία](/people-use-web/user-stories/#shopper)
```
Το ίδιο και εδώ να γίνουν update

Xrisanthi-Mil replied:
**final**
[Ian, υπάλληλος καταχώρισης δεδομένων με αυτισμό](/people-use-web/user-stories/story-two/)
[Lakshmi, ανώτερο στέλεχος - λογίστρια που είναι τυφλή](/people-use-web/user-stories/story-three/)
[Lexie, καταναλώτρια που πραγματοποιεί αγορές μέσω διαδικτύου που δυσκολεύεται να διακρίνει ορισμένα χρώματα (αχρωματοψία)](https://www.w3.org/WAI/people-use-web/user-stories/story-four/)
[Dhruv, μεγαλύτερος ενήλικας μαθητής που είναι κωφός](/people-use-web/user-stories/story-six/)
[Marta, βοηθός μάρκετινγκ που είναι κωφή και τυφλή](/people-use-web/user-stories/story-seven/)
[Elias, συνταξιούχος με περιορισμένη όραση, τρέμουλο στα χέρια και ήπιου βαθμού απώλεια βραχυπρόθεσμης μνήμης](/people-use-web/user-stories/story-nine/)

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+-   Όλη η λειτουργικότητα που είναι διαθέσιμη μέσω του ποντικιού είναι επίσης διαθέσιμη μέσω πληκτρολογίου
+-   Η εστίαση μέσω πληκτρολογίου δεν παγιδεύεται σε κανένα τμήμα του περιεχομένου
+-   Τα προγράμματα περιήγησης Παγκοσμίου Ιστού, τα εργαλεία συγγραφής και άλλα εργαλεία παρέχουν υποστήριξη μέσω πληκτρολογίου
```
να βγάλουμε το "παγκοσμιου ιστού" δεν χρειάζεται εδώ

Xrisanthi-Mil replied:
Accepted
Τα προγράμματα περιήγησης, τα εργαλεία συγγραφής και άλλα εργαλεία παρέχουν υποστήριξη μέσω πληκτρολογίου

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+-   Η εστίαση μέσω πληκτρολογίου δεν παγιδεύεται σε κανένα τμήμα του περιεχομένου
+-   Τα προγράμματα περιήγησης Παγκοσμίου Ιστού, τα εργαλεία συγγραφής και άλλα εργαλεία παρέχουν υποστήριξη μέσω πληκτρολογίου
+
+Η ικανοποίηση αυτής της απαίτησης βοηθά τους χρήστες πληκτρολογίου, συμπεριλαμβανομένων των ατόμων που χρησιμοποιούν εναλλακτικά πληκτρολόγια, όπως πληκτρολόγια με εργονομική διάταξη, πληκτρολόγια οθόνης ή συσκευές διακοπτών. Βοηθά επίσης άτομα που χρησιμοποιούν φωνητική αναγνώριση (εισαγωγή ομιλίας) να χειρίζονται ιστότοπους και να υπαγορεύουν κείμενο μέσω της διεπαφής του πληκτρολογίου.
```
Η εκπλήρωση αυτής της απαίτησης βοηθά τους χρήστες πληκτρολογίου, συμπεριλαμβανομένων των ατόμων που χρησιμοποιούν εναλλακτικά πληκτρολόγια, όπως πληκτρολόγια με εργονομικές διατάξεις, πληκτρολόγια οθόνης ή συσκευές switch.

Xrisanthi-Mil replied:
Accepted
Η εκπλήρωση αυτής της απαίτησης βοηθά τους χρήστες πληκτρολογίου, συμπεριλαμβανομένων των ατόμων που χρησιμοποιούν εναλλακτικά πληκτρολόγια, όπως πληκτρολόγια με εργονομική διάταξη, πληκτρολόγια οθόνης ή συσκευές διακοπτών (switch). Βοηθά επίσης άτομα που χρησιμοποιούν φωνητική αναγνώριση (εισαγωγή ομιλίας) να χειρίζονται ιστότοπους και να υπαγορεύουν κείμενο μέσω της διεπαφής του πληκτρολογίου.

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+{% include excol.html type="middle" %}
+
+-   [Alex, δημοσιογράφος με τραυματισμό επαναλαμβανόμενης καταπόνησης](/people-use-web/user-stories/#reporter)
```
Το ίδιο και εδώ

Xrisanthi-Mil replied:
[Ade, δημοσιογράφος με περιορισμένη χρήση των χεριών του](/people-use-web/user-stories/story-one/)
[Lakshmi, ανώτερο στέλεχος - λογίστρια που είναι τυφλή](/people-use-web/user-stories/story-three/)

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+Το καλά οργανωμένο περιεχόμενο βοηθά τους χρήστες να προσανατολιστούν μόνοι τους και να πλοηγηθούν αποτελεσματικά. Τέτοιο περιεχόμενο περιλαμβάνει:
+
+-   Οι σελίδες έχουν σαφείς τίτλους και είναι οργανωμένες με τη χρήση περιγραφικών επικεφαλίδων ενοτήτων
```
- με την χρήση περιγραφικών επικεφαλίδων στις ενότητες

Xrisanthi-Mil replied:
**final**
Οι σελίδες έχουν σαφείς τίτλους και είναι οργανωμένες με τη χρήση περιγραφικών επικεφαλίδων στις ενότητες

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+-   Υπάρχουν περισσότεροι από ένας τρόποι να βρεθούν σχετικές σελίδες στα πλαίσια ενός συνόλου ιστοσελίδων
+-   Οι χρήστες είναι ενήμεροι για την τρέχουσα τοποθεσία τους στα πλαίσια ενός συνόλου σχετικών σελίδων
+-   Υπάρχουν τρόποι να παρακαμφθούν τμήματα περιεχομένου που επαναλαμβάνονται σε πολλές σελίδες
+-   Η εστίαση του πληκτρολογίου είναι ορατή και η σειρά εστίασης ακολουθεί μια σειρά με νόημα
```
- ...και η σειρά της εστίασης ακολουθεί μια αλληλουχία με νόημα

Xrisanthi-Mil replied:
Accepted
Η εστίαση του πληκτρολογίου είναι ορατή και η και η σειρά της εστίασης ακολουθεί μια αλληλουχία με νόημα

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+-   Η εστίαση του πληκτρολογίου είναι ορατή και η σειρά εστίασης ακολουθεί μια σειρά με νόημα
+-   Ο σκοπός ενός συνδέσμου είναι προφανής, ιδανικά ακόμη και όταν ο σύνδεσμος προβάλλεται μόνος του
+
+Η ικανοποίηση αυτής της απαίτησης βοηθά τους ανθρώπους να πλοηγούνται στις ιστοσελίδες με διαφορετικούς τρόπους, ανάλογα με τις ιδιαίτερες ανάγκες και προτιμήσεις τους. Για παράδειγμα, ενώ μερικοί άνθρωποι βασίζονται σε ιεραρχικές δομές πλοήγησης, όπως γραμμές μενού για να βρουν συγκεκριμένες ιστοσελίδες, άλλοι αντ' αυτού βασίζονται σε λειτουργίες αναζήτησης στους ιστότοπους. Μερικοί άνθρωποι μπο ...
```
Η εκπλήρωση αυτής της απαίτησης βοηθά τους ανθρώπους να πλοηγούνται σε ιστοσελίδες με διαφορετικούς τρόπους, ανάλογα με τις ιδιαίτερες ανάγκες και προτιμήσεις τους. Για παράδειγμα, ενώ ορισμένοι βασίζονται σε ιεραρχικές δομές πλοήγησης, όπως είναι οι γραμμές μενού, για να βρουν συγκεκριμένες ιστοσελίδες, άλλοι βασίζονται σε λειτουργίες αναζήτησης σε ιστότοπους. Κάποιοι μπορεί να βλέπουν το περιεχόμενο, ενώ άλλοι μπορεί να το ακούν ή να το βλέπουν και να το ακούν ταυτόχρονα. Κάποιοι μπορεί να χρησιμοποιούν το περιεχόμενο μόνο με ποντίκι ή πληκτρολόγιο, ενώ άλλοι μπορεί να χρησιμοποιούν και τα δύο.

Xrisanthi-Mil replied:
Accepted

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+{% include excol.html type="middle" %}
+
+-   [Alex, δημοσιογράφος με τραυματισμό επαναλαμβανόμενης καταπόνησης](/people-use-web/user-stories/#reporter)
```
το ίδιο

Xrisanthi-Mil replied:
[Ade, δημοσιογράφος με περιορισμένη χρήση των χεριών του](/people-use-web/user-stories/story-one/)
[Ian, υπάλληλος καταχώρισης δεδομένων με αυτισμό](/people-use-web/user-stories/story-two/)
[Lakshmi, ανώτερο στέλεχος - λογίστρια που είναι τυφλή](/people-use-web/user-stories/story-three/)
[Sophie, φίλαθλος της καλαθοσφαίρισης με σύνδρομο Down](/people-use-web/user-stories/story-five/)
[Marta, βοηθός μάρκετινγκ που είναι κωφή και τυφλή](/people-use-web/user-stories/story-seven/)
[Stefan, μαθητής με διαταραχή ελλειμματικής προσοχής και υπερκινητικότητας, καθώς και με δυσλεξία](/people-use-web/user-stories/story-eight/)
[Elias, συνταξιούχος με περιορισμένη όραση, τρέμουλο στα χέρια και ήπιου βαθμού απώλεια βραχυπρόθεσμης μνήμης](/people-use-web/user-stories/story-nine/)

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+### Οι χρήστες μπορούν να χρησιμοποιήσουν διαφορετικές μεθόδους εισαγωγής δεδομένων εκτός του πληκτρολογίου {#modalities}
+
+Οι τρόποι εισαγωγής δεδομένων εκτός του πληκτρολογίου, όπως η ενεργοποίηση μέσω αφής, η αναγνώριση φωνής (εισαγωγή ομιλίας) και οι χειρονομίες κάνουν το περιεχόμενο πιο εύκολο στη χρήση από πολλούς ανθρώπους. Ωστόσο, δεν μπορούν όλοι να χρησιμοποιήσουν καθεμία από αυτές τις μεθόδους εισαγωγής δεδομένων και στον ίδιο βαθμό. Συγκεκριμένες σχεδιαστικές εκτιμήσεις μεγιστοποιούν το όφελος αυτών των τρ ...
```
- * όπως είναι η ενεργοποίηση μέσω αφής.
- * Ορισμένες σχεδιαστικές επιλογές

Xrisanthi-Mil replied:
**final**
Οι τρόποι εισαγωγής δεδομένων εκτός του πληκτρολογίου, όπως είναι η ενεργοποίηση μέσω αφής, η αναγνώριση φωνής (εισαγωγή ομιλίας) και οι χειρονομίες κάνουν το περιεχόμενο πιο εύκολο στη χρήση από πολλούς ανθρώπους. Ωστόσο, δεν μπορούν όλοι να χρησιμοποιήσουν καθεμία από αυτές τις μεθόδους εισαγωγής δεδομένων και στον ίδιο βαθμό. Ορισμένες σχεδιαστικές επιλογές μεγιστοποιούν το όφελος αυτών των τρόπων εισαγωγής δεδομένων. Αυτό περιλαμβάνει:

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+-   Η λειτουργικότητα που ενεργοποιείται με κίνηση μπορεί επίσης να επιτευχθεί μέσω των στοιχείων διεπαφής χρήστη
+-   Τα κουμπιά, οι σύνδεσμοι και άλλα ενεργά στοιχεία είναι αρκετά μεγάλα ώστε να είναι ευκολότερο να ενεργοποιηθούν μέσω αφής.
+
+Η ικανοποίηση αυτής της απαίτησης καθιστά το περιεχόμενο ευκολότερο στη χρήση για πολλούς ανθρώπους με ένα ευρύ φάσμα ικανοτήτων που χρησιμοποιούν ένα ευρύ φάσμα συσκευών. Αυτό περιλαμβάνει περιεχόμενο που χρησιμοποιείται σε κινητά τηλέφωνα, υπολογιστές tablet και τερματικά αυτοεξυπηρέτησης, όπως μηχανήματα έκδοσης εισιτηρίων.
```
Η εκπλήρωση αυτής της απαίτησης καθιστά το περιεχόμενο πιο εύχρηστο για πολλά άτομα με ευρύ φάσμα ικανοτήτων που χρησιμοποιούν ένα ευρύ φάσμα συσκευών. Αυτό περιλαμβάνει περιεχόμενο που χρησιμοποιείται σε κινητά τηλέφωνα, tablet και τερματικά αυτοεξυπηρέτησης, όπως μηχανήματα έκδοσης εισιτηρίων.

Xrisanthi-Mil replied:
accepted

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+{% include excol.html type="middle" %}
+
+-   [Alex, δημοσιογράφος με τραυματισμό επαναλαμβανόμενης καταπόνησης](/people-use-web/user-stories/#reporter)
```
Το ίδιο

Xrisanthi-Mil replied:
Ade, δημοσιογράφος με περιορισμένη χρήση των χεριών του (/people-use-web/user-stories/story-one/)
Sophie, φίλαθλος της καλαθοσφαίρισης με σύνδρομο Down(/people-use-web/user-stories/story-five/)
Stefan, μαθητής με διαταραχή ελλειμματικής προσοχής και υπερκινητικότητας, καθώς και με δυσλεξία (/people-use-web/user-stories/story-eight/)
Elias, συνταξιούχος με περιορισμένη όραση, τρέμουλο στα χέρια και ήπιου βαθμού απώλεια βραχυπρόθεσμης μνήμης (/people-use-web/user-stories/story-nine/)

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+###  Το κείμενο είναι ευανάγνωστο και κατανοητό {#readable}
+
+Οι συγγραφείς περιεχομένου πρέπει να διασφαλίζουν ότι το περιεχόμενο κειμένου είναι ευανάγνωστο και κατανοητό στο ευρύτερο δυνατό κοινό, συμπεριλαμβανομένης της ανάγνωσης του δυνατά από λογισμικό κειμένο σε ομιλία. Τέτοιο περιεχόμενο περιλαμβάνει:
```
- λογισμικό μετατροπής κειμένου σε ομιλία.

Xrisanthi-Mil replied:
**final**
Οι συγγραφείς περιεχομένου πρέπει να διασφαλίζουν ότι το περιεχόμενο κειμένου είναι ευανάγνωστο και κατανοητό στο ευρύτερο δυνατό κοινό, συμπεριλαμβανομένης της ανάγνωσης του δυνατά από λογισμικό μετατροπής κειμένου σε ομιλία. Τέτοιο περιεχόμενο περιλαμβάνει:

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+-   Παροχή ορισμών για τις ασυνήθιστες λέξεις, φράσεις, ιδιωματισμούς και συντομογραφίες
+-   Χρήση της σαφέστερης και απλούστερης δυνατής γλώσσας ή παροχή απλοποιημένων εκδόσεων
+
+Η ικανοποίηση αυτής της απαίτησης βοηθά το λογισμικό, συμπεριλαμβανομένης της υποστηρικτικής τεχνολογίας, να επεξεργάζεται σωστά το περιεχόμενο κειμένου. Για παράδειγμα, αυτή η απαίτηση βοηθά το λογισμικό να διαβάζει το περιεχόμενο δυνατά, να παράγει περιλήψεις σελίδων και να παρέχει ορισμούς για ασυνήθιστες λέξεις, όπως η τεχνική ορολογία. Βοηθά επίσης άτομα που δυσκολεύονται να κατανοήσουν πιο  ...
```
* εκπλήρωση

Xrisanthi-Mil replied:
Η εκπλήρωση αυτής της απαίτησης βοηθά το λογισμικό, συμπεριλαμβανομένης της υποστηρικτικής τεχνολογίας, να επεξεργάζεται σωστά το περιεχόμενο κειμένου. Για παράδειγμα, αυτή η απαίτηση βοηθά το λογισμικό να διαβάζει το περιεχόμενο δυνατά, να παράγει περιλήψεις σελίδων και να παρέχει ορισμούς για ασυνήθιστες λέξεις, όπως η τεχνική ορολογία. Βοηθά επίσης άτομα που δυσκολεύονται να κατανοήσουν πιο σύνθετες προτάσεις, φράσεις και λεξιλόγιο. Συγκεκριμένα, βοηθά ανθρώπους με διαφορετικούς τύπους γνωσιακών αναπηριών.

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+{% include excol.html type="middle" %}
+
+-   [Martine, βαρήκοη εξ' αποστάσεως φοιτήτρια](/people-use-web/user-stories/#onlinestudent)
```
το ίδιο

Xrisanthi-Mil replied:
[Ian, υπάλληλος καταχώρισης δεδομένων με αυτισμό](/people-use-web/user-stories/story-two/)
[Lakshmi, ανώτερο στέλεχος - λογίστρια που είναι τυφλή](/people-use-web/user-stories/story-three/)
[Sophie, φίλαθλος της καλαθοσφαίρισης με σύνδρομο Down](/people-use-web/user-stories/story-five/)
[Dhruv, μεγαλύτερος ενήλικας μαθητής που είναι κωφός](/people-use-web/user-stories/story-six/)
[Stefan, μαθητής με διαταραχή ελλειμματικής προσοχής και υπερκινητικότητας, καθώς και με δυσλεξία](/people-use-web/user-stories/story-eight/)
[Elias, συνταξιούχος με περιορισμένη όραση, τρέμουλο στα χέρια και ήπιου βαθμού απώλεια βραχυπρόθεσμης μνήμης](/people-use-web/user-stories/story-nine/)

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+-   Τα στοιχεία διεπαφής χρήστη που επαναλαμβάνονται σε ιστοσελίδες έχουν τις ίδιες ετικέτες κάθε φορά
+-   Οι σημαντικές αλλαγές σε μια ιστοσελίδα δεν πραγματοποιούνται χωρίς τη συγκατάθεση του χρήστη
+
+Η ικανοποίηση αυτής της απαίτησης βοηθά τους ανθρώπους να μάθουν γρήγορα τη λειτουργικότητα και τους μηχανισμούς πλοήγησης που παρέχονται σε έναν ιστότοπο και να τους λειτουργούν σύμφωνα με τις συγκεκριμένες ανάγκες και προτιμήσεις τους. Για παράδειγμα, κάποιοι άνθρωποι αναθέτουν εξατομικευμένα πλήκτρα συντόμευσης σε λειτουργίες που χρησιμοποιούν συχνά για να βελτιώσουν την πλοήγηση με πληκτρολόγ ...
```
* εκπλήρωση (παρακαλώ να αλλαχθεί σε όλα)

Xrisanthi-Mil replied:
**final**
Η εκπλήρωση αυτής της απαίτησης βοηθά τους ανθρώπους να μάθουν γρήγορα τη λειτουργικότητα και τους μηχανισμούς πλοήγησης που παρέχονται σε έναν ιστότοπο και να τους λειτουργούν σύμφωνα με τις συγκεκριμένες ανάγκες και προτιμήσεις τους. Για παράδειγμα, κάποιοι άνθρωποι αναθέτουν εξατομικευμένα πλήκτρα συντόμευσης σε λειτουργίες που χρησιμοποιούν συχνά για να βελτιώσουν την πλοήγηση με πληκτρολόγιο. Άλλοι απομνημονεύουν τα βήματα για να φτάσουν σε συγκεκριμένες σελίδες ή να ολοκληρώσουν διαδικασίες σε έναν ιστότοπο. Και οι δύο βασίζονται σε προβλέψιμη και συνεπή λειτουργικότητα.

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+
+{% include excol.html type="middle" %}
+
+-   [Alex, δημοσιογράφος με τραυματισμό επαναλαμβανόμενης καταπόνησης](/people-use-web/user-stories/#reporter)
```
Παρακαλώ να ανενεωθούν σε όλα

Xrisanthi-Mil replied:
[Ade, δημοσιογράφος με περιορισμένη χρήση των χεριών του](/people-use-web/user-stories/story-one/)
[Ian, υπάλληλος καταχώρισης δεδομένων με αυτισμό](/people-use-web/user-stories/story-two/)
[Lakshmi, ανώτερο στέλεχος - λογίστρια που είναι τυφλή](/people-use-web/user-stories/story-three/)
[Sophie, φίλαθλος της καλαθοσφαίρισης με σύνδρομο Down](/people-use-web/user-stories/story-five/)
[Marta, βοηθός μάρκετινγκ που είναι κωφή και τυφλή](/people-use-web/user-stories/story-seven/)
[Stefan, μαθητής με διαταραχή ελλειμματικής προσοχής και υπερκινητικότητας, καθώς και με δυσλεξία](/people-use-web/user-stories/story-eight/)
[Elias, συνταξιούχος με περιορισμένη όραση, τρέμουλο στα χέρια και ήπιου βαθμού απώλεια βραχυπρόθεσμης μνήμης](/people-use-web/user-stories/story-nine/)

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+Οι φόρμες και άλλες αλληλεπιδράσεις μπορεί να προκαλούν σύγχυση ή δυσκολία στη χρήση για πολλούς ανθρώπους και, ως αποτέλεσμα, μπορεί να είναι πιο πιθανό αυτοί να κάνουν λάθη. Παραδείγματα που βοηθούν τους χρήστες να αποφύγουν και να διορθώσουν λάθη περιλαμβάνουν:
+
+-   Περιγραφικές οδηγίες, μηνύματα λάθους και προτάσεις για διόρθωση
+-   Βοήθεια ευαίσθητη ως προς το γενικότερο πλαίσιο για πιο σύνθετη λειτουργικότητα και αλληλεπίδραση
```
- Σχετική βοήθεια ως προς το γενικότερο πλαίσιο

Xrisanthi-Mil replied:
**Final**
Οι φόρμες και άλλες αλληλεπιδράσεις μπορεί να προκαλούν σύγχυση ή δυσκολία στη χρήση για πολλούς ανθρώπους και, ως αποτέλεσμα, μπορεί να είναι πιο πιθανό αυτοί να κάνουν λάθη. Παραδείγματα που βοηθούν τους χρήστες να αποφύγουν και να διορθώσουν λάθη περιλαμβάνουν:

-   Περιγραφικές οδηγίες, μηνύματα λάθους και προτάσεις για διόρθωση
-   Σχετική βοήθεια ως προς το γενικότερο πλαίσιο για πιο σύνθετη λειτουργικότητα και αλληλεπίδραση

remibetin replied:
I've applied the change. See https://github.com/w3c/wai-website/commit/df533c0d156f047a535e7528b8fe0fffeff3a8c5

Please let me know if something is wrong.

avourliotakis commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -0,0 +1,724 @@
+-   Διασφάλιση ότι η σήμανση μπορεί να ερμηνευτεί αξιόπιστα, για παράδειγμα διασφαλίζοντας ότι είναι έγκυρη
+-   Παροχή ονόματος, ρόλου και τιμής για τα μη τυπικά στοιχεία διεπαφής χρήστη
+
+Η ικανοποίηση αυτής της απαίτησης βοηθά στη μεγιστοποίηση της συμβατότητας με τους τρέχοντες και τους μελλοντικούς πράκτορες χρήστη, συμπεριλαμβανομένων των υποστηρικτικών τεχνολογιών. Συγκεκριμένα, επιτρέπει στις υποστηρικτικές τεχνολογίες να επεξεργάζονται το περιεχόμενο αξιόπιστα και να το παρουσιάζουν ή να το χειρίζονται με διαφορετικούς τρόπους. Αυτό περιλαμβάνει μη τυπικά (σεναριακά) κουμπι ...
```
Με τις παραπάνω αλλαγές νομίζω ότι το κείμενο είναι σε ένα αξιόλογο επίπεδο. Πάντοτε υπάρχει χώρος για βελτίωση, αλλά προς το παρόν είμαστε οκ.

Xrisanthi-Mil replied:
@remibetin this is the final comment from @avourliotakis 
"With the above changes, I think the text is at a very good level. There is always room for improvement, but for the time being, we're good."

remibetin replied:
@Xrisanthi-Mil @avourliotakis Thank you for your collaborative work on this!

remibetin committed:
Apply final updates, provided by @Xrisanthi-Mil

remibetin committed:
Minor fixes

- Remove draft status
- Identify the language of English words
- Update last updated dates

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -142,12 +142,12 @@ footer: >
+- [[Dhruv, μεγαλύτερος ενήλικας μαθητής που είναι κωφός]]()
+- [[Marta, βοηθός μάρκετινγκ που είναι κωφή και τυφλή]](/people-use-web/user-stories/story-seven/)
+- [[Stefan, μαθητής με διαταραχή ελλειμματικής προσοχής και υπερκινητικότητας, καθώς και με δυσλεξία]](/people-use-web/user-stories/story-eight/)
+- [[Elias, συνταξιούχος με περιορισμένη όραση, τρέμουλο στα χέρια και ήπιου βαθμού απώλεια βραχυπρόθεσμης μνήμης]](/people-use-web/user-stories/story-nine/)
```
In these 6 rows there is double [[ and double ]]

remibetin replied:
Indeed, thanks for noticing! The latest version uses double brackets.

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -191,9 +191,11 @@ footer: >
+- [[Dhruv, μεγαλύτερος ενήλικας μαθητής που είναι κωφός]](/people-use-web/user-stories/story-six/)
+- [[Marta, βοηθός μάρκετινγκ που είναι κωφή και τυφλή]](/people-use-web/user-stories/story-seven/)
+
+
```
In these 3 rows there is double [[ and double ]]

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -241,13 +243,12 @@ footer: >
+- [[Dhruv, μεγαλύτερος ενήλικας μαθητής που είναι κωφός]](/people-use-web/user-stories/story-six/)
+- [[Marta, βοηθός μάρκετινγκ που είναι κωφή και τυφλή]](/people-use-web/user-stories/story-seven/)
+- [[Stefan, μαθητής με διαταραχή ελλειμματικής προσοχής και υπερκινητικότητας, καθώς και με δυσλεξία]](/people-use-web/user-stories/story-eight/)
+- [[Elias, συνταξιούχος με περιορισμένη όραση, τρέμουλο στα χέρια και ήπιου βαθμού απώλεια βραχυπρόθεσμης μνήμης]](/people-use-web/user-stories/story-nine/)
```
In these 6 rows there is double [[ and double ]]

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -300,11 +304,12 @@ footer: >
+- [[Lexie, καταναλώτρια που πραγματοποιεί αγορές μέσω διαδικτύου που δυσκολεύεται να διακρίνει ορισμένα χρώματα (αχρωματοψία)]](/people-use-web/user-stories/story-four/)
+- [[Dhruv, μεγαλύτερος ενήλικας μαθητής που είναι κωφός]](/people-use-web/user-stories/story-six/)
+- [[Marta, βοηθός μάρκετινγκ που είναι κωφή και τυφλή]](/people-use-web/user-stories/story-seven/)
+- [[Elias, συνταξιούχος με περιορισμένη όραση, τρέμουλο στα χέρια και ήπιου βαθμού απώλεια βραχυπρόθεσμης μνήμης]](/people-use-web/user-stories/story-nine/)
```
In these 6 rows there is double [[ and double ]]

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -353,8 +358,8 @@ footer: >
--   [Ilya, ανώτερο στέλεχος προσωπικού που είναι τυφλή](/people-use-web/user-stories/#accountant)
+- [[Ade, δημοσιογράφος με περιορισμένη χρήση των χεριών του]](/people-use-web/user-stories/story-one/)
+- [[Lakshmi, ανώτερο στέλεχος - λογίστρια που είναι τυφλή]](/people-use-web/user-stories/story-three/)
 
```
In these 6 rows there is double [[ and double ]]

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -486,26 +491,27 @@ footer: >
+- [[Sophie, φίλαθλος της καλαθοσφαίρισης με σύνδρομο Down]](/people-use-web/user-stories/story-five/)
+- [[Marta, βοηθός μάρκετινγκ που είναι κωφή και τυφλή]](/people-use-web/user-stories/story-seven/)
+- [[Stefan, μαθητής με διαταραχή ελλειμματικής προσοχής και υπερκινητικότητας, καθώς και με δυσλεξία]](/people-use-web/user-stories/story-eight/)
+- [[Elias, συνταξιούχος με περιορισμένη όραση, τρέμουλο στα χέρια και ήπιου βαθμού απώλεια βραχυπρόθεσμης μνήμης]](/people-use-web/user-stories/story-nine/)
```
In these 7 rows there is double [[ and double ]]

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -525,24 +531,25 @@ footer: >
+- [[Ade, δημοσιογράφος με περιορισμένη χρήση των χεριών του]](/people-use-web/user-stories/story-one/)
+- [[Sophie, φίλαθλος της καλαθοσφαίρισης με σύνδρομο Down]](/people-use-web/user-stories/story-five/)
+- [[Stefan, μαθητής με διαταραχή ελλειμματικής προσοχής και υπερκινητικότητας, καθώς και με δυσλεξία]](/people-use-web/user-stories/story-eight/)
+- [[Elias, συνταξιούχος με περιορισμένη όραση, τρέμουλο στα χέρια και ήπιου βαθμού απώλεια βραχυπρόθεσμης μνήμης]](/people-use-web/user-stories/story-nine/)
```
In these 4 rows there is double [[ and double ]]

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -568,11 +575,12 @@ footer: >
+- [[Sophie, φίλαθλος της καλαθοσφαίρισης με σύνδρομο Down]](/people-use-web/user-stories/story-five/)
+- [[Dhruv, μεγαλύτερος ενήλικας μαθητής που είναι κωφός]](/people-use-web/user-stories/story-six/)
+- [[Stefan, μαθητής με διαταραχή ελλειμματικής προσοχής και υπερκινητικότητας, καθώς και με δυσλεξία]](/people-use-web/user-stories/story-eight/)
+- [[Elias, συνταξιούχος με περιορισμένη όραση, τρέμουλο στα χέρια και ήπιου βαθμού απώλεια βραχυπρόθεσμης μνήμης]](/people-use-web/user-stories/story-nine/)
```
In these 6 rows there is double [[ and double ]]

Xrisanthi-Mil commented on pages/fundamentals/accessibility-principles/index.el.md:
```diff
@@ -615,12 +623,13 @@ footer: >
+- [[Sophie, φίλαθλος της καλαθοσφαίρισης με σύνδρομο Down]](/people-use-web/user-stories/story-five/)
+- [[Marta, βοηθός μάρκετινγκ που είναι κωφή και τυφλή]](/people-use-web/user-stories/story-seven/)
+- [[Stefan, μαθητής με διαταραχή ελλειμματικής προσοχής και υπερκινητικότητας, καθώς και με δυσλεξία]](/people-use-web/user-stories/story-eight/)
+- [[Elias, συνταξιούχος με περιορισμένη όραση, τρέμουλο στα χέρια και ήπιου βαθμού απώλεια βραχυπρόθεσμης μνήμης]](/people-use-web/user-stories/story-nine/)
```
In these 7 rows there is double [[ and double ]]

remibetin committed:
Apply final updates, provided by @Xrisanthi-Mil

remibetin committed:
Update translators
